servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DelegatingListenableAsyncCloseable.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2025 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.concurrent.api;
+
+/**
+ * {@link ListenableAsyncCloseable} that delegates all calls to another {@link ListenableAsyncCloseable}.
+ */
+public class DelegatingListenableAsyncCloseable extends DelegatingAsyncCloseable implements ListenableAsyncCloseable {
```

Comment:
Do you want to implement a `toString` here as well?

Reply:
The inheritance from `DelegatingAsyncCloseable ` will do the trick of producing the same outcome